partcad/src/partcad/part_factory_sweep.py

```
@@ -0,0 +1,178 @@
+
+            self._create(config)
+
+    async def instantiate(self, part):
+        with pc_logging.Action("Sweep", part.project_name, part.name):
+            shape = None
+            try:
+                self.sketch = self.project.ctx.get_sketch(self.source_sketch_spec)
+
+                # Convert path points to TColgp_Array1OfPnt
+                from OCP.TColgp import TColgp_Array1OfPnt
+                from OCP.TColStd import TColStd_Array1OfReal
```

Ruff (0.8.2)

`OCP.TColStd.TColStd_Array1OfReal` imported but unused

Remove unused import: `OCP.TColStd.TColStd_Array1OfReal`

(F401)